File: src/main/java/historywowa/domain/heritage/application/service/HeritageService.java
package historywowa.domain.heritage.application.service;

import historywowa.domain.heritage.presentation.dto.res.HeritageImageRes;
import java.io.IOException;
import org.springframework.web.multipart.MultipartFile;

public interface HeritageService {

    HeritageImageRes recognize(MultipartFile heritageImage, String userId) throws IOException;
}

File: src/main/kotlin/historywowa/domain/heritage/application/service/HeritageService.kt
package historywowa.domain.heritage.application.service

import historywowa.domain.heritage.presentation.dto.res.HeritageImageRes
import org.springframework.web.multipart.MultipartFile

interface HeritageService {
    fun recognize(heritageImage: MultipartFile, userId: String): HeritageImageRes
}
